Hotfix/30 prop types on screens

OwenRamire opened the pull request:
closes #30 
# Description
Using PropTypes on the components check if they are receiving all the props that they required. And when we call at the components not everyone has all the props that they need, so the JS console sends a warning that says that we are not passing the required props.
This Pull Request fix those warnings on the console

OwenRamire committed:
Fix PropTypes on components

The components had the keyword propType and the JS console send a
warning that tell if we mean proptype to validate props and in the
components validate those proptypes wrong

OwenRamire committed:
Pass all required props

LeyaimMagma commented on src/screens/RegisterAccount.js:
```diff
@@ -22,17 +22,20 @@ const RegisterAccount = ({navigation}) => {
             title="Correo electrónico"
```
Add this on Constants.js

LeyaimMagma commented on src/screens/ContactInformation.js:
```diff
@@ -29,6 +29,7 @@ const ContactInformation = () => {
             title={element}
             isRequired="Recomendado"
```
Add this on Constants.js

LeyaimMagma commented on src/screens/RegisterAccount.js:
```diff
@@ -22,17 +22,20 @@ const RegisterAccount = ({navigation}) => {
+            keyboard="email-address"
           />
           <EmailPassTxtInput
             title="Clave"
```
Add this on Constants.js

fernandoamz replied:
@LeyaimMagma, Correct me if this is not true. But, If He adds this to the constant file he will get an error because this is a `prop` from the keyboard

LeyaimMagma commented on src/screens/signInScreens/ResetPassConfirm.js:
```diff
@@ -60,12 +60,14 @@ const ResetPassConfirm = ({route}) => {
             title="Nueva clave"
```
Add this on Constants.js

LeyaimMagma commented on src/screens/signInScreens/ResetPassConfirm.js:
```diff
@@ -60,12 +60,14 @@ const ResetPassConfirm = ({route}) => {
           />
           <PasswordRequirements />
           <EmailPassTxtInput
             title="Escribe de nueva clave"
```
Add this on Constants.js

OwenRamire committed:
Update constants

Add the keyboard type object with properties to store the differents
keyboards that RN has

OwenRamire committed:
Use constant messages

call the properties that have the keyboard object and some other
constants messages
